### DISCUSSION TITLE
Export-DbaLogin takes long time to run

### DISCUSSION THREAD

**Discussion by 0x7FFFFFFFFFFFFFFF** [dataplat/dbatools#8550] [Q&A]
I'm completely new to dbatools and it seems very interesting. I tried several commands and all work well. However, when I try the command `Export-DbaLogin`, it takes ages to run and never finishes and I have to stop it. Why does this happen? Is it because some of my logins are created from domain accounts?

**Comment by potatoqualitee** [comment]
I've seen this and it usually has to do with a DC with high latency or some sort of timeout issue 🤔

**Reply by 0x7FFFFFFFFFFFFFFF** [reply]
If the login is from a domain account, it just need to generate something like `CREATE LOGIN [<domainName>\<login_name>] FROM WINDOWS;`. Why need to access the DC?

**Reply by niphlod** [reply]
yeah. how many database and jobs are on the instance ? having tons may have an impact because of fetching the mapping between users and logins. Try running the command passing -ExcludeDatabase and -ExcludeJobs
